fluss-common/src/main/java/org/apache/fluss/metrics/MetricNames.java

```diff
@@ -132,6 +132,8 @@ public class MetricNames {
     public static final String LOG_NUM_SEGMENTS = "numSegments";
     public static final String LOG_END_OFFSET = "endOffset";
     public static final String REMOTE_LOG_SIZE = "size";
+    public static final String LOG_LAKE_LAG_RECORDS = "lakeLagRecords";
```

Comment:
I don't have a good naming in my mind, maybe

```suggestion
// for lake tiering
    public static final String LOG_LAKE_PENDING_RECORDS = "pendingRecords";
```

Inspired from Flink https://cwiki.apache.org/confluence/display/FLINK/FLIP-33%3A+Standardize+Connector+Metrics